PSReadLine/ReadLine.vi.cs

```diff
@@ -515,7 +515,15 @@ private void ViIndicateCommandMode()
         {
             if (_options.ViModeIndicator == ViModeStyle.Cursor)
             {
-                _console.CursorSize = _normalCursorSize < 50 ? 100 : 25;
+                if (PlatformWindows.IsConsoleApiAvailable())
+                {
+                    _console.CursorSize = _normalCursorSize < 50 ? 100 : 25;
+                }
+                else
+                {
+                    // Set the cursor to a solid blinking block.
+                    Console.Write("\x1b[2 q");
```

Comment:
This probably belongs in the console wrapper.

To help unit testing, I don't use any `Console` apis directly, if I do, point them out so I can fix.

By wrapping all uses of the `Console` api, I also stand a chance at getting this stuff working on Win7/Win8 where there is no support for escape sequences unless you have something like ConEmu.

Reply:
I'll move it.